hooks/useWalletProvider.tsx

```diff
@@ -100,11 +100,11 @@ const useWalletProvider = () => {
       // modal, as "User closed modal"
       console.log('error', e)
     }
-  }, [handleAccountsChanged, web3Modal])
+  }, [])
 
   useEffect(() => {
-    const infuraId =
-      process.env.NEXT_PUBLIC_INFURA_ID || 'b6058e03f2cd4108ac890d3876a56d0d'
+    const infuraId = getInfuraId()
```

Comment:
Is this defined somewhere that I'm missing? Don't see any import or reference to it in the file